kolibri/utils/constants/installation_types.py

```diff
@@ -0,0 +1,24 @@
+"""
+This module contains constants representing the type of "installers" used to install Kolibri.
+"""
+from __future__ import unicode_literals
+
+APK = "apk"
+APT = "apt"
+KOLIBRI_SERVER = "kolibri-server"
+MACOS = "Mac"
+PEX = "pex"
+UWSGI = "uwsgi"
+WHL = "whl"
+WINDOWS = "Windows"
+
+install_type_map = {
+    APK: "APK",
```

Comment:
It helps with having this centrally documented/enforced, so the various installers don't go and do their own things, yeah.

Main thing is that it'd need to be considered a template/prefix, rather than a simple equivalence match.